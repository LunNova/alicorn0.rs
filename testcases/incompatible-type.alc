

let footype = record
	x : host-number

let foo = the footype record-of(x = "x")
