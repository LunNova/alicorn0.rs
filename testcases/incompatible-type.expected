{ { "let", "footype", "=", "record", { "x", ":", "host-number" } }, { "let", "foo", "=", "the", "footype", { "record-of", { "x", "=", { elements = { "x" }, kind = "string" } } } } }
